backend/grant/task/views.py

```diff
@@ -0,0 +1,19 @@
+from datetime import datetime
+
+from flask import Blueprint, jsonify
+from grant.task.jobs import JOBS
+from grant.task.models import Task, tasks_schema
+
+blueprint = Blueprint("task", __name__, url_prefix="/api/v1/task")
+
+
+@blueprint.route("/", methods=["GET"])
+def task():
+    tasks = Task.query.filter(Task.execute_after <= datetime.now()).all()
```

Comment:
Should this also filter on `Task.completed`?

Reply:
good catch! I've added tests for this too now.